index.js

```diff
@@ -70,6 +71,25 @@ module.exports = {
       });
   },
 
+  addOrientationDegreesChangeListener(callback) {
+    var key = getKey(callback);
+    listeners[key] = DeviceEventEmitter.addListener(orientationDegreesDidChangeEvent,
+      (body) => {
+        callback(body.orientationDegrees);
+      });
+  },
+
+  removeOrientationDegreesChangeListener(callback) {
```

Comment:
I'm probably going to delete this because with the current approach, removing a JS listener doesn't stop the event from getting sent over the react native bridge. If there's a way to stop sending events over the bridge, then it'd make more sense to remove the listener sometimes